src/helpers/useSubscription.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+/**
+ * useSubscription makes it easy to make a simple event emitter.
+ * @returns {object} An object with functions `subscribe` and `emit`. The
+ * `subscribe` method accepts a callback function and returns an `unsubscribe`
+ * function, and `emit` will broadcast a value to all subscribers
+ */
+export const useSubscription = () => {
```

Comment:
I lowkey love little subscriber patterns like this.
```js
const unsubscribe = subscribe(newValue => doSomethingWith(newValue))
```
This is a handy dandy little hook for quickly creating them.